.gitmodules

```diff
@@ -5,3 +5,4 @@
 [submodule "test/tests"]
 	path = test/jsontests
 	url = https://github.com/ethereum/tests.git
+  branch = invalid-ops
```

Comment:
@jwasinger you could just do the following

cd cpp-ethereum/test/jsontests
git checkout invalid-ops

and then git for cpp-ethereum will automatically assign subrepo to a new branch